Assets/HoloToolkit/Input/Scripts/InputManager.cs

```diff
@@ -388,10 +388,10 @@ public void HandleEvent<T>(BaseEventData eventData, ExecuteEvents.EventFunction<
                 handler.OnInputClicked(casted);
             };
 
-        public void RaiseInputClicked(IInputSource source, uint sourceId, InteractionSourcePressInfo pressType, int tapCount, object tag = null)
+        public void RaiseInputClicked(IInputSource source, uint sourceId, InteractionSourcePressInfo pressType, int tapCount, object[] tags = null)
         {
             // Create input event
-            sourceClickedEventData.Initialize(source, sourceId, tag, pressType, tapCount);
+            sourceClickedEventData.Initialize(source, sourceId, tags, pressType, tapCount);
```

Comment:
Should we move the tags parameter to the end of the statements that use it and make it null by default as well to keep it compatible with the master branch?